lib/tasks/i18n.rake

```diff
@@ -0,0 +1,15 @@
+namespace :i18n do
+  require 'cbx_loco'
+
+  desc "Extract i18n assets, and upload them to the online service"
```

Comment:
I know this was moved from the original code, but I think that since we labelled this "cbx_loco", we could say "upload them to Loco" instead of being vague and saying "the online service".

There are 4 references to "online service" in total. I recommend adapting them all to say "Loco" instead.